packages/jade/tests/match.html

```diff
@@ -118,4 +118,9 @@
 <template name="match-html-specialMarkdownComponent">{{#markdown}}This is my email: <address@example.com>
 {{/markdown}}</template>
 
-<template name="match-html-specialMarkdownComponentWithHelper">{{#markdown}}This is my email: {{email}}{{/markdown}}</template>
+<template name="match-html-specialMarkdownComponentWithHelper">{{#markdown}}This is my email: {{email}}
+{{/markdown}}</template>
+
+<template name="match-html-anonymousIfHelper">{{#if _jade_line257}}hello world{{/if}}</template>
```

Comment:
Relying on the generated helper name (here `_jade_line257`) is kind of fragile because (the line number might change for example) and I think we shouldn’t rely on it. (How do you handle 2 anonymous helpers on the same line BTW?)

I we should create a second tester function in `match.js` that would abstract the anonymous helpers and consider the test successful for whatever helper name we have as long as the helper itself and its surrounding template is correct.